apps/labs/posts/lets-make-pygments-accessible.md

```diff
@@ -0,0 +1,70 @@
+- Saturation, or “how much of that color is present?” You can also ask yourself if it looks like it’s mixed with other colors, white, black, or grey. ⚖️
+
+The most common issues in syntax highlighting—where each color has a certain meaning—will be sufficient color contrast (an issue of value) and using distinct hues. 
+
+The themes we’ve added to `accessible-pygments` so far follow [WCAG AA color contrast requirements](https://www.w3.org/TR/WCAG22/#contrast-minimum). This means that there is a good amount of difference in value between a given color and the other colors it overlaps with, usually the background color. Because syntax highlighting is mostly text based, we need to follow guidelines for text contrast at normal text sizes. There are a few ways you could review this, but a quick way to dive in is to plug colors into an [online contrast checker](https://webaim.org/resources/contrastchecker) like this one.
+
+Distinct hues refers to reviewing that themes maintain their visible differences even if their colors cannot be fully perceived. For example, a theme using all different shades of blue would have no distinct hues; they are all blue! If you have limited vision, they may all blend together like there is no syntax highlighting at all. Colors that are close together on the color wheel can also be difficult for some people to distinguish. For example, blue and purple can easily look like one color. But if someone decides to lean into their differences—like a true blue and a more red purple— these hues may be easier to distinguish. A good example of a distinct color combination would be blue and orange. Not only are these opposite hues from each other on the color wheel, they have different innate values, and do not have large amounts of red or green (hues that can be especially tricky to distinguish in the most common cases of color blindness). For now, the authors are not aware of a good single test for ensuring that hues are distinct. Sometimes color blindness simulators, like the [A11y Color Blindness Empathy browser extension](https://vinceumo.github.io/A11Y-Color-Blindness-Empathy-Test/), can help you pinpoint these issues when you turn on a simulation.
+
+The best accessibility experiences are often ones with strong defaults that welcome customization. In `accessible-pygments`, we are still focused on the strong defaults portion of that statement. Because we do not support customization, our goal is to [provide a growing list of curated accessible themes](https://github.com/Quansight-Labs/accessible-pygments/issues/2) for people to choose from. Having options so that everyone can choose what works best for them allows people to have agency and limit how disabling their environment is.
+
+As a note: general accessibility advice is to not rely exclusively on color to convey any information. Since syntax highlighting is all color information the authors have yet to find if there is an alternate approach.
```

Comment:
But syntax highlighting doesn't convey any information that isn't already in the text itself. All it does it make the code easier to read by distinguishing the different types of code tokens.

Reply:
This is true! No information is lost by not being able to make use of syntax highlighting for any reason, but the kind of support that it can provide for someone reading code is lost as far as I'm aware. I probably didn't make that distinction clear enough.

Do you want me to remove the note in line 46?

Reply:
Agree - no information is lost - _what does get affected though_ is more aligned with readability/scannability (if that is a word). 
Though - some IDE's and syntax highlighters _do_ allow the use of _italics_ to complement the "conveying information with colour"